lib/hanami/action/cache/conditional_get.rb

```diff
@@ -67,7 +67,10 @@ def initialize(env, value)
         # @since 0.3.0
         # @api private
         def fresh?
-          !Hanami::Utils::Blank.blank?(modified_since) && Time.httpdate(modified_since).to_i >= @value.to_time.to_i
+          return false if Hanami::Utils::Blank.blank?(modified_since)
+          return false if Hanami::Utils::Blank.blank?(@value)
```

Comment:
Hey, not sure that I'm understand correctly your tests but could you add test case for this line too?

Reply:
This is covered by my specs, if you remove the diff you'll notice it 500s without this line

Reply:
Awesome! Thanks for explanation.